packages/create-toolpad-app/index.ts

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env node
+
+import chalk from 'chalk';
+import { execaCommand } from 'execa';
+import inquirer from 'inquirer';
+import fs from 'fs';
+import { getPackageManager } from './utils/getPackageManager';
+
+// Detect the package manager
+const packageManager = getPackageManager();
+
+// Create a new directory and initialize a new project
+const scaffoldProject = async () => {
+  const name = await inquirer.prompt([
+    {
+      type: 'input',
+      name: 'projectName',
+      message: 'Enter the name of your project:',
+      default: 'my-app',
+    },
+  ]);
+
+  // eslint-disable-next-line no-console
+  console.log(`Creating a new MUI Toolpad project in ${chalk.blue(name.projectName)}`);
+  try {
+    fs.mkdirSync(name.projectName);
```

Comment:
We can use async methods by importing `import * as fs from 'fs/promises';`
Probably wouldn't make a huge difference here but I guess it's more consistent with how we're using it in other files.